src/ios/GoogleService-Info.plist

```diff
@@ -19,7 +19,7 @@
 	<key>PLIST_VERSION</key>
```

Comment:
I think this file can be removed completely for now. It's for integrating firebase with an iOS application.